src/Executor.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // Executor This struct ensures the parallel execution of all command executions
 type Executor struct {
-	planets []Planet
+	planets []*Planet
```

Comment:
If not done this way, changes at SSH-Handler level of the scope wouldn't be persistent. E.G. if execCommand set the errored flag of the planet to true, it would still be false at executor-level scope. (Most interestingly, it's no pure call by value/reference issue, as the changes are persistent up to an inconsistent number of scopes...maybe it's a goroutine issue?)